client/src/main/java/agolf/lobby/LobbySinglePlayerPanel.java

```diff
@@ -196,9 +210,9 @@ public void actionPerformed(ActionEvent evt) {
                                         .getSelectedIndex() /*+ (isUsingCustomServer ? ("\t" + this.choicerTrackCategory.getSelectedIndex()) : "")*/);
             } else {
                 if (var2 == this.buttonStartChampionship) {
```

Comment:
Why not deobfuscase `var2`, too?

I guess that would be `eventSource` or similar.

Reply:
Fixed.